src/jest.config.ts

```diff
@@ -31,6 +31,10 @@ export default {
   testEnvironment: 'node',
   reporters: [
     'default',
-    ['jest-junit', { outputDirectory: 'results', outputName: 'report.xml' }]
+    ['jest-junit', { outputDirectory: 'results', outputName: 'report.xml' }],
+    ["./node_modules/jest-html-reporter", {
+      "pageTitle": "BAV Test Report",
+      "outputPath": "results/test-report.html"
```

Comment:
<img width="1440" alt="Screenshot 2023-11-30 at 10 23 52" src="https://github.com/govuk-one-login/ipv-cri-bav-api/assets/147814888/c1d99c54-0781-41d8-8a4b-ddc9bf3d7160">

Reply:
Nice yeah I guess what I'm asking is why do we need to generate this report? Do QA use it regularly, is it needed in some sort of automation?

Reply:
Its more of a 'nice-to-have' feature, it helps visualise test results where before we couldn't - it won't have any impact on tests. QA will have the ability to use it as much as they need/like

Reply:
Cool, is it something that we want to save somewhere then? I can see that we archive the lint results, do we want to do the same with this?

Reply:
I dont believe we'd need to archive an results of this, I can think of very few situations that would benefit from doing so. But it may not hurt to add?

Reply:
ok let's proceed with this for now and if we find a need later we can add the step to the workflow 👍